<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleFileResourceConfigWidget</class>
 <widget class="QWidget" name="SingleFileResourceConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>0</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QGroupBox" name="pathGroupBox">
     <property name="title">
      <string comment="@title:group">Calendar File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="pathLabel">
          <property name="text">
           <string comment="@label:textbox">Filename:</string>
          </property>
          <property name="buddy">
           <cstring>pathRequester</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="pathRequester"/>
        </item>
        <item>
         <widget class="QLineEdit" name="pathText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string notr="true">Status:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="KTextEdit" name="pathDescription">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="html">
         <string>Select the file whose contents should be represented by this resource. If the file does not exist, it will be created. The URL of a remote file can also be specified, but note that monitoring for file changes will not work in this case.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="alarmTypeGroupBox">
     <property name="title">
      <string comment="@title:group">Alarm Type</string>
     </property>
     <layout class="QVBoxLayout" name="gridLayout">
      <item>
       <layout class="QHBoxLayout" name="layout">
        <item>
         <widget class="QRadioButton" name="activeRadio">
          <property name="text">
           <string comment="@option:radio">Acti&amp;ve Alarms</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">alarmTypeGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="archivedRadio">
          <property name="text">
           <string comment="@option:radio">Archived Alarms</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">alarmTypeGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="templateRadio">
          <property name="text">
           <string comment="@option:radio">Alarm &amp;Templates</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">alarmTypeGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="KTextEdit" name="alarmTypeDescription">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="html">
         <string>Select which alarm type this resource should contain.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="alarmTypeLabel">
          <property name="text">
           <string comment="@label:textbox">Alarm type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="activeAlarmsText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="@item">Active Alarms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="archivedAlarmsText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="@item">Archived Alarms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="templateAlarmsText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="@item">Alarm Templates</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="nameGroupBox">
     <property name="title">
      <string comment="@title:group">Display Name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string comment="@label:textbox">&amp;Name:</string>
          </property>
          <property name="buddy">
           <cstring>displayNameText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="displayNameText"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="KTextEdit" name="nameDescription">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="html">
         <string>Enter the name used to identify this resource in displays.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="readOnlyGroupBox">
     <property name="title">
      <string comment="@title:group">Access Rights</string>
     </property>
     <layout class="QVBoxLayout" name="gridLayout_4">
      <item>
       <widget class="QCheckBox" name="readOnlyCheckbox">
        <property name="text">
         <string comment="@option:check">Read only</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KTextEdit" name="readOnlyDescription">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="html">
         <string>If read-only mode is enabled, no changes will be written to the file selected above. Read-only mode will be automatically enabled if you do not have write access to the file or the file is on a remote server that does not support write access.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="alarmTypeGroup"/>
 </buttongroups>
</ui>
